toolset/utils/docker_helper.py

```diff
@@ -11,6 +11,10 @@
 from toolset.utils.output_helper import log
 from toolset.utils.database_helper import test_database
 
+from psutil import virtual_memory
+
+# total memory limit allocated for the test container
+mem_limit = virtual_memory().total * .95
```

Comment:
Running inside a docker container doesn't have trouble with this?

Reply:
No, currently the docker containers have all the virtual memory of the host